packages/fluentui/react-bindings/src/styles/resolveStyles.ts

```diff
@@ -41,7 +43,6 @@ const stylesCache = new WeakMap<ThemePrepared, Record<string, ICSSInJSStyle>>();
 const resolveStyles = (
   options: ResolveStylesOptions,
   resolvedVariables: ComponentVariablesObject,
-  renderStylesInput?: (styles: ICSSInJSStyle) => string,
```

Comment:
This was used only for tests actually, but as we already using `renderRule` from `options.renderer` I decided to remove it to simplify the logic.